### ISSUE TITLE
Install CaC on disconnected cluster

### ISSUE THREAD

**Issue by david-rh** [redhat-cop/ocp-disconnected-docs#35]
Document the process to install and use Compliance as Code on a disconnected cluster.

**Comment by arvin-a** [comment]
What is the current issue with running CaC in disconnected clusters?

**Comment by ppetko** [comment]
Any updates on this? Is this related to the compliance operator?

**Comment by david-rh** [comment]
Yes, this is the compliance operator. I think this was on hold waiting til oc-mirror was readily available. Then document the process for using oc-mirror to deploy CaC into a disconnected cluster.
